src/js/controllers/chatController.js

```diff
@@ -1,3 +1,4 @@
+/* global Promise:false */
```

Comment:
I don't make the rules, jshint is :p ... I'll move that to the jshint config instead.